README.md

```diff
@@ -39,6 +39,12 @@ for the repository that will be auto-detected when you open the folder in VSCode
 
 This should™ also be supported by Github Codespaces, but we haven't tested it.
 
+Make sure you have [the Remote Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)
+and [Docker Desktop](https://www.docker.com/products/docker-desktop) installed.
```

Comment:
Could be fairly important to mention *not* enabling Docker Desktop to autolaunch and to not forget to close docker after you're done working on the docs. Docker Desktop uses WSL which will guzzle tons of memory and not give it back until shut down